### DISCUSSION TITLE
Welcome to DataGridXL2 Discussions!

### DISCUSSION THREAD

**Discussion by DataGridXL** [DataGridXL/DataGridXL2#96] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DataGridXL** [comment]
@manuelderuiter... test! Manuel, wil je laten weten of je een notificatie van deze comment hebt ontvangen? Ik zoek een manier om de stargazers van DGXL2 te benaderen.

**Reply by manuelderuiter** [reply]
Yes, works!

**Comment by DataGridXL** [comment]
@datagridxl laat ik het ook met mezelf proberen

**Comment by DataGridXL** [comment]
@DataGridXL laat ik het nog eens proberen
